modules/nodes-nodes-audit-log-basic-viewing.adoc

```diff
@@ -32,28 +32,28 @@ ip-10-0-170-165.ec2.internal audit.log
 .. View the {product-title} API server log for a specific master node and timestamp or view all the logs for that master:
 +
 ----
-$ oc adm node-logs <node-ip> <log-name> --path=openshift-apiserver/<log-name>
+$ oc adm node-logs <node-ip> --path=openshift-apiserver/<log-name>
```

Comment:
Due to above comment, pls change the PR's all occurrences of `oc adm node-logs <node-ip>` to `oc adm node-logs <node-name>`